fly.toml

```
@@ -8,6 +8,9 @@ console_command = "/rails/bin/rails console"
 primary_region = "ams"
 swap_size_mb = 512
 
+[env]
+  RAILS_SERVE_STATIC_FILES = "1"
+
 [build]
 
 [deploy]
```

After applying the changes I suggested, I assume that the deploy section should look like this:
```suggestion
[deploy]
  release_command = "bundle exec rails runner scripts/deploy/post-deploy"
```